cmd/container-disk-v1alpha/entry-point.sh

```diff
@@ -40,6 +40,26 @@ IMAGE_EXTENSION=$(echo $IMAGE_PATH | sed  -n -e 's/^.*\.\(.*\)$/\1/p')
 mkdir -p $COPY_PATH
 echo $IMAGE_NAME | grep -q -e "raw" -e "qcow2"
 if [ $? -ne 0 ]; then
+	output=$(/usr/bin/qemu-img info $IMAGE_PATH --output=json)
+	if [ $? -ne 0 ]; then
+        echo "qemu-img info returned error"
+        exit 1
+    fi
+
+    IFS=","
+    echo $output | while read -r LINE; do
+    #it is not a valid image if its format is not qcow2.
+    if [ `echo $LINE | awk '$1 == "\"format\":" {print $1 }'` ]  && [ `echo $LINE | awk '$2 != "\"qcow2\"" {print $2 }'` ] ; then
```

Comment:
this pr limited conversion only for qcow. I'm pointing out that because this error checking didn't actually cause the script exit early that .iso files are still being converted to .raw.

our functional tests depend on an alpine .iso container disk. Those tests are still working becaues this pr did not actually limit conversion to just .qcow images.

Reply:
Added code to exit from the script when we break from the loop because of an error.
Also changed the indention.